Tuist/Interfaces/UIKit/Sources/Data/NetworkAPI/Core/NetworkAPIProtocol.swift

```diff
@@ -3,12 +3,14 @@
 //
 
 import Alamofire
-import RxAlamofire
-import RxSwift
+import Combine
```

Comment:
@nkhanh44 Could you check how much effort you need to implement `async/await` for `NetworkAPI`, just want to minimize the framework in this layer

Reply:
@suho I updated it here https://github.com/nimblehq/ios-templates/pull/525/commits/670ef6bf55b77120df4fe930d6b108e4edaf17d3 🙏  Please help to review again